hack/multi-node/user-data.sample

```diff
@@ -25,13 +25,16 @@ coreos:
       content: |
         [Service]
         EnvironmentFile=/etc/environment
-        Environment=KUBELET_ACI=quay.io/coreos/hyperkube
-        Environment=KUBELET_VERSION=v1.3.0-alpha.5_coreos.0
+        Environment=KUBELET_ACI=quay.io/peanutbutter/hyperkube
+        Environment=KUBELET_VERSION=v1.3.0-alpha.5_derek.1
```

Comment:
Drop for now until we have official image / we need the fixes contained here explicitly